postgres/pgdump.go

```diff
@@ -265,7 +265,7 @@ func processCreateStmt(conv *internal.Conv, n nodes.CreateStmt) {
 			// Note: there should be at most one Constraint node in
 			// n.TableElts.Items. We don't check this. We just keep
 			// collecting constraints.
-			constraints = extractConstraints(conv, n, table, []nodes.Node{i})
```

Comment:
This seems like it was a bug in the previous code -- can you confirm?

Also, is the comment (the previous three lines) relevant anymore?

Reply:
This is the pgdump approach and it was not developed by me so I am not sure.

Reply:
Comments -- I don't suppose it's much relevant, but it would be good to just keep it.